WD-Firmware/rtos/rtosal/src/rtosal_time.c

```diff
@@ -26,11 +26,13 @@
 * include files
 */
 #include "rtosal_time_api.h"
-#include "rtosal_macro.h"
+#include "rtosal_macros.h"
 #include "rtosal.h"
 #include "psp_api.h"
 #ifdef D_USE_FREERTOS
    #include "timers.h"
```

Comment:
check if needed: timers.h

Reply:
It is required as it calls timers.h APIs